user-guide/docs/tools/simulation.md

```diff
@@ -8,6 +8,8 @@
 
 {% include-markdown 'simulation/dakota.md' %}
 
+{% include-markdown 'simulation/in-core.md' %}
```

Comment:
@silviamazzoni, should this content be included on a page that is accessible via the navigation?

I can see the content on http://localhost:8000/user-guide/tools/simulation/, but that page does not show in nav.